features/shipping/managing_shipments/access_shipments_order_from_shipment_index.feature

```diff
@@ -0,0 +1,22 @@
+@managing_shipments
+Feature: Access shipment's order from the shipments index
+    In order to make shipments and orders management more fluent
+    As an Administrator
+    I want to be able to access shipment's page from shipments index
```

Comment:
`I want to be able to access shipment's page from shipments index` -> `I want to be able to access order's page from shipments index`